util/jsonutil/jsonutil.go

```diff
@@ -124,6 +126,11 @@ var jsonConfigValidationOff = jsoniter.Config{
 	ValidateJsonRawMessage: false,
 }.Froze()
 
+func init() {
+	jsonConfigValidationOff.RegisterExtension(&RawMessageExtension{})
+	jsonConfigValidationOn.RegisterExtension(&RawMessageExtension{})
+}
```

Comment:
My untested attempt at an init func. Doesn't work for tests, but I think that's what [TestMain](https://medium.com/goingogo/why-use-testmain-for-testing-in-go-dafb52b406bc) is for.

Reply:
Added a `TestMain` per every `*_test.go` file that needs to register the extension